addons/resource/models/resource.py

```diff
@@ -333,65 +333,82 @@ def _check_attendance(self):
     # --------------------------------------------------
     # Computation API
     # --------------------------------------------------
-    def _attendance_intervals(self, start_dt, end_dt, resource=None, domain=None, tz=None):
+    def _attendance_intervals(self, start_dt, end_dt, resources=None, domain=None, tz=None):
         """ Return the attendance intervals in the given datetime range.
             The returned intervals are expressed in specified tz or in the resource's timezone.
         """
+        self.ensure_one()
+        if resources is None:
+            resources = self.env['resource.resource']
         assert start_dt.tzinfo and end_dt.tzinfo
         combine = datetime.combine
 
-        resource_ids = [resource.id, False] if resource else [False]
+        resources_list = [self.env['resource.resource']]
+        if resources:
+            resources_list = [r for r in resources] + resources_list
```

Comment:
```suggestion
        resources_list = [r for r in resources] + [self.env['resource.resource']]
```

Reply:
but is it expected that the last item in an empty record? My understanding is that we just wanted to add `False` in `resource_ids` for below domain, not keep it in the for loop below, can't we just keep `resources` as it and add `False` in the domain?